pymysql/connections.py

```diff
@@ -1191,4 +1227,70 @@ def _get_descriptions(self):
         assert eof_packet.is_eof_packet(), 'Protocol error, expecting EOF'
         self.description = tuple(description)
 
+
+class LoadLocalFile(object):
+    def __init__(self, filename, connection):
+        self.filename = filename
+        self.connection = connection
+
+    def _print_warnings(self):
+        self.connection._execute_command(COMMAND.COM_QUERY, 'SHOW WARNINGS')
+        warnings = MySQLResult(self.connection)
+        warnings.read()
```

Comment:
Remove this method.  Utility method like this should implemented by user or separated function.
Since `print()` may fails, protocol implementation should not print anything.